nc_time_axis/__init__.py

```diff
@@ -274,20 +279,20 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
-        if not isinstance(first_value, (CalendarDateTime, cftime.datetime)):
+        if not isinstance(first_value, (CalendarDateTime, cftime_datetime)):
             raise ValueError('The values must be numbers or instances of '
                              '"nc_time_axis.CalendarDateTime" or '
-                             '"cftime.datetime".')
+                             '"cftime datetime".')
 
         if isinstance(first_value, CalendarDateTime):
-            if not isinstance(first_value.datetime, cftime.datetime):
+            if not isinstance(first_value.datetime, cftime_datetime):
                 raise ValueError('The datetime attribute of the '
                                  'CalendarDateTime object must be of type '
-                                 '`cftime.datetime`.')
+                                 '`cftime.datetime_base`.')
```

Comment:
This would be better using `type(cftime_datetime)` or similar to construct the `ValueError` message.